### PR TITLE
chore(UIKIT-4248): update design details

### PR DESCRIPTION
**Pull request by bang9**
## For Internal Contributors

[UIKIT-4248]

## Description Of Changes

- separate `ParentMessage` component into a separate file
- apply parent message composition to the `GroupChannelMessage` component
- update some design details
- add exception handling for unreachable parent message

## Types Of Changes

What types of changes does your code introduce to this project?
Put an `x` in the boxes that apply_

- [ ] Bugfix
- [ ] New feature
- [ ] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance (ex) Prettier)
- [ ] Build configuration
- [x] Improvement (refactor code)
- [ ] Test

[UIKIT-4248]: https://sendbird.atlassian.net/browse/UIKIT-4248?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by bang9** [comment]
If everything looks good, please feel free to merge this 🙇‍♂️

File: packages/uikit-utils/src/sendbird/message.ts [hunk 1]
```diff
@@ -117,6 +118,19 @@ export function parseSendbirdNotification(dataPayload: RawSendbirdDataPayload):
...
 }
 
+export function shouldRenderParentMessage(message: SendbirdMessage): message is (
+  | SendbirdUserMessage
+  | SendbirdFileMessage
+) & {
+  parentMessage: SendbirdUserMessage | SendbirdFileMessage;
+} {
+  if (message.isFileMessage() || message.isUserMessage()) {
+    if (!message.parentMessage) return false;
```
Comment (chrisallo) [inline packages/uikit-utils/src/sendbird/message.ts:128]: 이거 message.parentMessage?.isFileMessage() || message.parentMessage?.isUserMessage()로 퉁칠 수 있지 않을까요?

Comment (bang9) [reply packages/uikit-utils/src/sendbird/message.ts:128]: 오 네네 if 문이라 가능할거같습니당

**Review by chrisallo — APPROVED** [review]
lgtm!

**Pull request merged**